lib/utils/api_client.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:logger/logger.dart';
+import 'package:provider/provider.dart';
+
+import '../configs/configs.dart';
+import '../configs/end_point_urls.dart';
+import '../constants/app_constants.dart';
+import '../constants/strings.dart';
+import 'http_client.dart';
+import '../providers/page.dart';
+import '../providers/user.dart';
+import '../providers/user_session.dart';
+import '../utils/auth_client.dart';
+import '../utils/util.dart';
+
+final logger = Logger();
+
+class APIClient{
```

Comment:
Better to move the files related to API / https to another folder, for better management.

Reply:
fixed